apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.js

```diff
@@ -2,32 +2,35 @@
 const asyncHandler = require('express-async-handler');
 import * as express from 'express';
 import * as path from 'path';
+import { z } from 'zod';
 
 import { flash } from '@prairielearn/flash';
-import * as sqldb from '@prairielearn/postgres';
+import { loadSqlEquiv, queryRow, queryRows } from '@prairielearn/postgres';
 import { logger } from '@prairielearn/logger';
 import * as Sentry from '@prairielearn/sentry';
 
 import * as opsbot from '../../lib/opsbot';
 import * as github from '../../lib/github';
 import { config } from '../../lib/config';
+import { CourseRequestSchema, IdSchema } from '../../lib/db-types';
 
 const router = express.Router();
-const sql = sqldb.loadSqlEquiv(__filename);
-
-router.use((req, res, next) => {
-  res.locals.navPage = 'request_course';
-  next();
-});
+const sql = loadSqlEquiv(__filename);
 
 router.get(
   '/',
   asyncHandler(async (req, res) => {
-    const result = await sqldb.queryAsync(sql.get_requests, {
-      user_id: res.locals.authn_user.user_id,
-    });
+    res.locals.navPage = 'request_course';
 
-    res.locals.course_requests = result.rows;
+    const result = await queryRows(
+      sql.get_requests,
+      {
+        user_id: res.locals.authn_user.user_id,
+      },
+      CourseRequestSchema,
+    );
+
+    res.locals.course_requests = result;
```

Comment:
Nit:

```suggestion
    res.locals.course_requests = await queryRows(
      sql.get_requests,
      {
        user_id: res.locals.authn_user.user_id,
      },
      CourseRequestSchema,
    );
```

Reply:
I like that better! Addressed in cf09e7d